### PR TITLE
Update events.md

### PR DESCRIPTION
**Pull request by jessstarr-okta**
## Description:
- Changing wording on Events API Documentation page to reflect deprecation details. 
- Is this PR related to a Monolith release? No

### Resolves:

* [OKTA-204137](https://oktainc.atlassian.net/browse/OKTA-204137)

### CODE REVIEW

**Comment by jakubvul** [comment]
@jessstarr-okta are you also going to add a note [here](https://developer.okta.com/use_cases/events-api-migration/)?

File: _source/_docs/api/resources/events.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ redirect_from: "/docs/api/rest/events.html"
 
 The Okta Events API provides read access to your organization's system log. [Export event data](https://support.okta.com/help/Documentation/Knowledge_Article/Exporting-Okta-Log-Data) as a batch job from your organization to another system for reporting or analysis.
 
-> **Important:** the [System Log API](/docs/api/resources/system_log) will eventually replace the Events API and contains much more [structured data](/docs/api/resources/system_log#logevent-object). Developers of new projects are strongly recommended to use the System Log API in lieu of the Events API. For information on migrating from the Events API to the System Log API please see [Events API Migration](/use_cases/events-api-migration/).
+> **Important:** the [System Log API](/docs/api/resources/system_log) will eventually replace the Events API and contains much more [structured data](/docs/api/resources/system_log#logevent-object). As of Jan 7, 2018 developers of new projects will be unable to access the Events API and should use the System Log API. For information on migrating from the Events API to the System Log API please see [Events API Migration](/use_cases/events-api-migration/).
```
Comment (johngronberg-okta) [inline _source/_docs/api/resources/events.md]: "Jan 7, 2018" ... "will be"  

should this be in a different tense? Also is 2018 correct or should it read 2019?

Comment (jakubvul) [reply _source/_docs/api/resources/events.md]: It should be "are unable". And yes, definitely 2019.

Comment (jessstarr-okta) [reply _source/_docs/api/resources/events.md]: Yes to both, changes made.

**Pull request merged**